server/pulp/server/db/model/__init__.py

```diff
@@ -315,6 +317,7 @@ class TaskStatus(AutoRetryDocument, ReaperMixin):
     start_time = ISO8601StringField()
     finish_time = ISO8601StringField()
     result = DynamicField()
+    group_id = StringField(default=None)
```

Comment:
Should this be a UUIDField instead?

http://docs.mongoengine.org/en/latest/apireference.html#mongoengine.fields.UUIDField